arbitrum-docs/node-running/node-providers.mdx

```diff
@@ -26,6 +26,8 @@ This section provides an overview of the available public RPC endpoints for diff
 
 ⚠️ Unlike the RPC Urls, the Sequencer endpoints only support `eth_sendRawTransaction` and `eth_sendRawTransactionConditional` calls.
 
+⚠️ Important note: WebSocket calls are not supported when using the Arbitrum public endpoints.
+
 ⚠️ Important note: Ethereum Goerli was deprecated in Q1 2023 but will be supported until Q4 2023. Once Arbitrum Sepolia becomes publicly available, we will phase out the usage of Arbitrum Goerli. Support for Arbitrum Goerli will cease prior to the conclusion of Ethereum Goerli.
```

Comment:
```suggestion
⚠️ Important note: Ethereum Goerli was deprecated in Q1 2023 but will be supported until Q1 2024. Once Arbitrum Sepolia becomes publicly available, we will phase out the usage of Arbitrum Goerli. Support for Arbitrum Goerli will cease prior to the conclusion of Ethereum Goerli.
```
I think? Not 100% sure.

Reply:
I think that's correct (based on a little search). But let's see if @amarrazza can confirm.

Reply:
@TucksonDev I'll commit your suggestion and resolve the convo as I'm 98% sure it's correct